app/src/main/java/org/schabi/newpipe/player/VideoPlayerImpl.java

```diff
@@ -880,9 +880,11 @@ private void onMoreOptionsClicked() {
 
     private void onShareClicked() {
         // share video at the current time (youtube.com/watch?v=ID&t=SECONDS)
+        // Timestamp doesn't make sense in a live stream so drop it
+        String timestamp = !isLive() ? "&t=" + getPlaybackSeekBar().getProgress() / 1000 : "";
```

Comment:
Could you make this variable `final`? Could you also reverse the ternary statement so that it isn't negative, i.e. `isLive ? "" : "&t=" + …`?

Thanks for your contribution.

Reply:
Done!